### ISSUE TITLE
Replace Escalation in Core

### ISSUE THREAD

**Issue by millerj-tc** [ACG8/Lucid#186]
For the second test in a row, I've had a newer player make mistakes using Escalation. In this case, Olivia spent 3/5 on the first two abilities and I had to point out that she could use 5/5 by using the second two abilities instead. Anna made a similar mistake during the last test. I think those costs are hard to parse for new players and we should replace Escalation with a two tiered dream.

Since we are adding in a 4 cost and a 5 cost card, I think we should have something that rewards you for reaching 6 rather than a scaling card since that feels less impactful. The player who puts in the time to get 6 should feel like they can do something no one else can do.

![image](https://user-images.githubusercontent.com/3642851/43679312-ca593b9a-97e8-11e8-98e4-0269e6503cf8.png)

![image](https://user-images.githubusercontent.com/3642851/43679326-1fac95d8-97e9-11e8-8343-cce4f7889ee3.png)

![image](https://user-images.githubusercontent.com/3642851/43679362-89792f58-97e9-11e8-8752-b4c81319e7ae.png)

![image](https://user-images.githubusercontent.com/3642851/43679395-d7e66b60-97e9-11e8-8ea8-192030fe88aa.png)

![image](https://user-images.githubusercontent.com/3642851/43679407-2c4f6f62-97ea-11e8-8d21-227a317ffb5f.png)

I like the idea of having one more card draw card to help with Floof Sanctuary and all the bury costs in the game now.

**Comment by millerj-tc** [comment]
I also like the idea of having all the symbols in the core, hence the basic bury cost on one since Floof doesn't have that. It's annoying to have to pull out a draft card to point out non-core symbols.

**Comment by millerj-tc** [comment]
We could also put Bubblegum in the core as the last channel and then rework Fungal Discovery?

![image](https://user-images.githubusercontent.com/3642851/43679871-119df400-97f3-11e8-9314-9faf7f44bbd4.png)

I think this is still simpler than Escalation because it has one fewer ability and the costs are different which makes them easier to parse.

**Comment by ACG8** [comment]
Not saying that we shouldn't replace escalation with card draw, but:

Behold: Binary Escalation

![escalation](https://user-images.githubusercontent.com/10094820/43680976-e05db688-97fb-11e8-942e-711774727317.jpg)

If you use the abilities in order, you always get the maximum number of points possible. Also, it scales perfectly from 1 to 7 insight.

**Comment by millerj-tc** [comment]
Here's another way to fit in Bubblegum, include a bury cost in core, and a 6 drop:

![image](https://user-images.githubusercontent.com/3642851/43681087-4cb00ae4-980f-11e8-8503-0d4fe6d4c259.png)
